Azael/index.js

```diff
@@ -1,17 +1,52 @@
 // ### Basic Arithmetic Problems
 
 // Q1. In one night, a movie theater sells tickets for 6450 dollars. Each ticket costs 15 dollars. How many tickets were sold?
+let total = 6450;
+let singleTicket = 15;
+
+let totalTickets = total / singleTicket;
+
+console.log("total tickets sold==>" + totalTickets + " " + "dollars");
 
 // Q2. Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
 
+let incomeWeekly = 500;
+let weeksYear = 12 * 4;
```

Comment:
There is 52 weeks per year and not 48. Most of other students had the same answer.
Therefore you have to multiply 500 to 52

Reply:
Upsssss I will change it